### ISSUE TITLE
Cannot query for schema when running multiple dgraph nodes.

### ISSUE THREAD

**Issue by pawanrawal** [dgraph-io/dgraph#1620]
To reproduce

1. Start one Dgraphzero node.
2. Start two Dgraph nodes. 
3. Try to query for `schema{}` on any of the dgraph nodes. The error below is returned.

```
error while fetching schema error: internal error: Tablet isn't being served by this instance.
```

**Comment by pawanrawal** [comment]
This is happening because we blindly set schema for `_predicate_` on all nodes. We shouldn't do this and instead, propose a schema mutation for `_predicate_` if no other node is serving it.

**Comment by pawanrawal** [comment]
Merged a fix to both `master` and `release/v0.8.3`.
